whisper_online.py

```diff
@@ -4,6 +4,9 @@
 import librosa  
 from functools import lru_cache
 import time
+import logging
```

Comment:
#76 proposes
`logger = logging.getLogger(__name__)`

Can you check/decide what is better? What is a better practice, what are pros and cons? Is it easily configurable when imported as a module?

Reply:
I've done this in ebdde208 and it's idiomatic as far as the `logging` docs are concerned, but whether it's a good fit or not depends on what your feelings about `__name__` are. If you run it as `python whisper_online_server.py ...` then `__name__` is `"__main__"`.  If you import it as a module from another file, it's `"whisper_online_server"`.  Personally I find that a little confusing, but I may be an outlier here.  Either way, the patch is in.

And yes, it is configurable when imported as a module: that's how I'm setting the log level in `faster-whisper`.

Reply:
This is a good point. Both files are also meant to be used as Python scripts. Maybe you are right and it is better to set it explicitly.

(In "professional™ Python development" you would probably use entrypoints, but I think this project should stay simple.)